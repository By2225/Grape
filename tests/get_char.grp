fun Int main() {
    String a;
    a = "applejacks";
    prints(get_char(a, 5));
    return 0;
}
