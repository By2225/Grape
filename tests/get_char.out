././scripts/test: line 11: ././tests/get_char: No such file or directory
